flags/ask_password_unix.go

```diff
@@ -1,4 +1,4 @@
-// +build !windows
+// +build !wasm
```

Comment:
Why remove windows?

Reply:
Also what does `!` signify?

Reply:
`!` signifies `not` operation. The file will be excluded if it's a Windows build.  
there are other options like `and` and `or `

Reply:
Is this tested?